qiskit/visualization/pass_manager_visualization.py

```diff
@@ -193,3 +193,160 @@ def _get_node_color(pss, style):
             return color
 
     return "black"
+
+
+@_optionals.HAS_GRAPHVIZ.require_in_call
+@_optionals.HAS_PYDOT.require_in_call
+def staged_pass_manager_drawer(pass_manager, filename=None, style=None, raw=False):
+    """
+    Draws the staged pass manager.
+
+        This function needs `pydot <https://github.com/erocarrera/pydot>`, which in turn needs
+    Graphviz <https://www.graphviz.org/>` to be installed.
+
+    Args:
+        pass_manager (StagedPassManager): the staged pass manager to be drawn
+        filename (str): file path to save image to
+        style (dict or OrderedDict): keys are the pass classes and the values are
+            the colors to make them. An example can be seen in the DEFAULT_STYLE. An ordered
+            dict can be used to ensure a priority coloring when pass falls into multiple
+            categories. Any values not included in the provided dict will be filled in from
+            the default dict
+        raw (Bool) : True if you want to save the raw Dot output not an image. The
+            default is False.
+    Returns:
+        PIL.Image or None: an in-memory representation of the pass manager. Or None if
+        no image was generated or PIL is not installed.
+    Raises:
+        MissingOptionalLibraryError: when nxpd or pydot not installed.
+        VisualizationError: If raw=True and filename=None.
+
+    Example:
+        .. code-block::
+
+             %matplotlib inline
```

Comment:
I tried running this example but I got an error about a missing coupling map. I'm not very familiar with the V2 backends but is it possible there is something missing in your example?

Reply:
Good catch, need to actually call the `FakeLagosV2()` function when creating the pass manager :laughing:

Reply:
verified that the following works:

```python
from qiskit.providers.fake_provider import FakeLagosV2
from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager

pass_manager = generate_preset_pass_manager(3, FakeLagosV2())
pass_manager.draw()
```